server/cli/cli.go

```diff
@@ -43,6 +43,9 @@ Output build version information.
 		fmt.Fprintf(w, "Build Time:  %s\n", info.Time)
 		fmt.Fprintf(w, "Build Deps:\n\t%s\n",
 			strings.Replace(strings.Replace(info.Deps, " ", "\n\t", -1), ":", "\t", -1))
+		if err := w.Flush(); err != nil {
+			fmt.Println(err)
```

Comment:
nit:

either `_ = w.Flush()`
or
```
if err := w.Flush(); err != nil {
    panic(err.Error())
}
```

unless there's a way to return an honest exit code to the shell.